includes/class-wc-payments-customer-service.php

```diff
@@ -241,6 +241,36 @@ public function get_payment_methods_for_customer( $customer_id, $type = 'card' )
 		}
 	}
 
+	/**
+	 * Get customer's saved payment methods that are enabled by store.
+	 *
+	 * @param string $customer_id Customer id.
+	 *
+	 * @return array
+	 * @throws \WCPay\Exceptions\API_Exception
+	 */
+	public function retrieve_usable_customer_payment_methods ( $customer_id ) {
```

Comment:
This will retrieve all of the customer's payment methods. However, merchants have the ability to enable or disable specific payment methods, rendering some of them unavailable. For instance, if a customer adds a payment method using SEPA DEBIT, and the merchant deactivates that option, the chosen payment method could become invalid.